import lib

try:
    print(test(1,2,3))
except NameError:
    print('Could not be resolved')
#end


